style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+  padding: 0;
+  outline: none;
+}
+
+/* Start of header */
+.header {
+  left: 0;
+  top: 0;
+  right: 0;
+  height: 60px;
+  width: 100vw;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
```

Comment:
- [ ] The contents of your header, in your case the logo and the nav, should be aligned using flexbox. Kindly implement that. Please also remember you that first of all, you need to change the `div` element that you are currently using for the header, and use the `header` element for it.

Reply:
I have wrapped:
1. header in a header tag 
2.  welcome in an anchor and out of nav 
3.  used flexbox for the header
Additionally, I have implemented all the optional requirements.